cookbooks/cdo-varnish/templates/default/default.vcl.erb

```diff
@@ -77,42 +77,56 @@ sub vcl_recv
   }
 ###
 
-  # Normalize Accept-Language header into X-Varnish-Accept-Language for better cache hit rates.
-  # To vary content based on the language, set Vary: X-Varnish-Accept-Language in application code.
-  C{
-      vcl_rewrite_accept_language(ctx);
-  }C
-
-### EXTRACT COOKIE HEADERS
+### REQUEST COOKIE+HEADER FILTER
 if(req.http.Cookie) {
   cookie.parse(req.http.Cookie);
-<% node['cdo-varnish']['cookie_headers'].to_hash.each do |cookie, header| %>
-  if(cookie.isset("<%= cookie %>")) {
-    set req.http.<%= header %> = cookie.get("<%= cookie %>");
-    cookie.delete("<%= cookie %>");
+}
+<%= config = node['cdo-varnish']['config']
```

Comment:
Is the thing that gets printed the value of second statement rather than first?  Also a bit surprising to rely on a side effect in a print statement.

This is confusing.  Could you just put the first line with the side effect in plain old <% %>?